ddl/column_change_test.go

```diff
@@ -118,7 +118,7 @@ func TestColumnAdd(t *testing.T) {
 		switch job.SchemaState {
 		case model.StateWriteOnly:
 			writeOnlyTable = tbl
-		case model.StatePublic:
+		case model.StateNone:
```

Comment:
Why change this state? It seems that we don't change `add column` related schema states.

Reply:
Good catch, it is a mistake.